webapp/loggedin/modules/designer/surveyInfo/components/srsEditor.js

```diff
@@ -21,7 +21,7 @@ const SrsEditor = props => {
     <InputChips
       itemsLookupFunction={srsLookupFunction}
       itemKeyProp="code"
-      itemLabelProp="name"
+      itemLabelFunction={srs => `${srs.name} (EPSG:${srs.code})`}
```

Comment:
should we use SRS.getName and SRS.getCode ?

Reply:
right